tests/migration/online_migration/zypper_patch.pm

```diff
@@ -18,6 +18,7 @@ use version_utils qw(is_desktop_installed is_sles4sap);
 use migration;
 use qam;
 use Utils::Backends 'is_pvm';
+use bootloader_setup;
```

Comment:
include the used function 'change_grub_config' is enough.

Reply:
changed.